### ISSUE TITLE
Front-end: Create front-end environment

### ISSUE THREAD

**Issue by clayhan** [technocore-redux/youcanhireus#1]
1. Figure out how to create environment, possibly without NPM.

**Comment by franklee77** [comment]
Yeah, so we totally have to use NPM! Which is fine tho - there's a package called python-webpack-loader that integrates both libraries.

So front-end wise, we can just proceed as if this whole thing is running in Node!

**Comment by clayhan** [comment]
I'll load in a skeleton and we can decipher if we would like to keep it!
